File: create_array.py
import numpy as np
import random
import string

# Criar um array multidimensional (3x4x5)
array = np.random.rand(3, 4, 5)

# Gerar nome aleatório para o arquivo
nome_aleatorio = ''.join(random.choices(string.ascii_letters + string.digits, k=10))
nome_arquivo = f"{nome_aleatorio}.npy"

# Salvar o array
np.save(nome_arquivo, array)

print(f"Array multidimensional criado com dimensões: {array.shape}")
print(f"Arquivo salvo como: {nome_arquivo}")

File: create_array.jl
# Criar um array multidimensional (3x4x5)
array = rand(3, 4, 5)

# Gerar nome aleatório para o arquivo
nome_arquivo = string(randstring(10), ".jld2")

# Salvar o array usando JLD2
using JLD2
save(nome_arquivo, "array", array)

println("Array multidimensional criado com dimensões: ", size(array))
println("Arquivo salvo como: ", nome_arquivo)
